new-app/src/App/App.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
-import RunGuess from './runGame';
+import RunGame from './runGame';
```

Comment:
It's convention for files that export components to be in Pascale case:

`import RunGame from "./RunGame"`